src/sklearn_evaluation/plot/classification.py

```diff
@@ -206,6 +210,9 @@ def confusion_matrix(
 
     Examples
     --------
+
+    This example illustrates how to plot a confusion matrix for binary classifier
```

Comment:
please make descriptions shorter, add a colon at the end, and check any missing prepositions

example. from:

```
This example illustrates how to plot a confusion matrix for binary classifier
```

to:

```
Plot a confusion matrix for a binary classifier:
```


apply the same for the other descriptions